### PR TITLE
New middle gray preview background color and vertical togglebuttons

### PR DESCRIPTION
**Pull request by Beep6581**

### CODE REVIEW

**Comment by Beep6581** [comment]
See https://github.com/Beep6581/RawTherapee/issues/4027

**Comment by TooWaBoo** [comment]
The gray icon should have a dark border. It's hard to see on brighter themes.

unbenannt

File: rtgui/indclippedpanel.cc [hunk 1]
```diff
@@ -34,39 +34,39 @@ IndicateClippedPanel::IndicateClippedPanel (ImageArea* ia) : imageArea(ia)
...
     Glib::ustring tt;
 
-    indclippedh = Gtk::manage (new Gtk::ToggleButton ());
-    indclippedh->set_relief(Gtk::RELIEF_NONE);
-    indclippedh->add (*Gtk::manage (new RTImage ("warnhl.png")));
-    tt = Glib::ustring::compose("%1\n%2 = %3", M("MAIN_TOOLTIP_INDCLIPPEDH"), M("MAIN_TOOLTIP_THRESHOLD"), options.highlightThreshold);
+    indClippedH = Gtk::manage (new Gtk::ToggleButton ());
+    indClippedH->set_relief(Gtk::RELIEF_NONE);
+    indClippedH->add (*Gtk::manage (new RTImage ("warnhl.png")));
+    tt = Glib::ustring::compose("%1\n%2 = %3", M("MAIN_TOOLTIP_indClippedH"), M("MAIN_TOOLTIP_THRESHOLD"), options.highlightThreshold);
```
Comment (Hombre57) [inline rtgui/indclippedpanel.cc]: Language strings should remain all uppercase.

Comment (Beep6581) [reply rtgui/indclippedpanel.cc]: Yeah, that was unintended.

**Pull request merged**